### ISSUE TITLE
Add support for tagging

### ISSUE THREAD

**Issue by riker09** [projecthamster/hamster-gtk#76]
In the original Hamster app it was possible to categorize items by a simple tagging feature. Will there be something similar for hamster-gtk? I did a quick glimpse at the app, but was not able to find something alike.

I think tags are a great way to keep tracked items organized. Plus if I ever wanted to automatically log my tracked times to Jira, Redmine, etc. a tag with the issue key would be very helpful.

**Comment by elbenfreund** [comment]
Yes. I personally do not use tags but since its last release ``hamster-lib`` actually has support for them. ``hamster-gtk`` currently does not do anything with them though and there is no representation for them at all.
Future versions of ``hamster-gtk`` (probably not the very next one though) will provide support for tags, so keep an eye out for them :)
